### DISCUSSION TITLE
Nvdia xorg.conf not loading

### DISCUSSION THREAD

**Discussion by SunnyOd** [leftwm/leftwm#721] [Q&A]
Hey Guys,

I'm running Manjaro Gnome with LeftWM installed over it. Running all the latest updates with newest nvidia drivers. I've got a 3 monitor setup with one of the monitors being a 144hz gaming monitor. Issue I'm seeing is that Gnome seems to remember the 144hz refresh rate when I log in but LeftWM doesn't. 

I've tried running nvidia-settings as root and saving the config file to /etc/X11/xorg.confd/20-nvidia.conf, and also /etc/X11/xorg.conf but when I restart LeftWM it reverts back to 60hz. Maybe I need to put an xorg.conf somewhere else? 

Btw. Just wanna say thank you for the work you guys are doing. I've just switched over to tiling window managers for the first time a week or so ago. After trying a bunch of the "best ones" I ended up pulling my hair out trying to get everything smooth but then found LeftWM. Love the project and the work is really coming along at pace. Looking forward to supporting in any way I can

Cheers!
Sunny

**Comment by SunnyOd** [comment]
Fixed... I found an articles which explains how many tiling window managers don't support nvidia xorg.conf files and to use xrandr... I managed to get it working using an up script in leftwm directory..

**Reply by VuiMuich** [reply]
Welcome adventurer!
Glap you got it fixed, thanks for reporting anyways, this might help someone else.

**Reply by SunnyOd** [reply]
Hi! Thank for the warm welcome, VuiMuich :)

Well looks like i was wrong abotu the ~/.config/leftwm/up script starting xrandr, I hadn't commented out the .desktop file ! 

Just trying to get the leftwm/up script working but I'm not sure if I'm doing it correctly. Does this look okay?

##
~/.config/leftwm >>> ls -l up
.rwxr-xr-x sunny sunny 659 B Mon Feb 28 18:03:28 2022  up
#
~/.config/leftwm >>> cat up
#!/bin/bash
xrandr --output DP-2 --mode 2560x1440 --rate 144.00
/usr/lib/polkit-gnome/polkit-gnome-authentication-agent-1 & & eval $(gnome-keyring-daemon -s --components=pkcs11,secrets,ssh,gpg)
echo `date` > ~/.config/leftwm/last-time-master-up-invoked.out

Thanks!
Sunny
